src/main/java/com/powsybl/openloadflow/network/impl/LfStaticVarCompensatorImpl.java

```diff
@@ -24,23 +24,21 @@
 
     private final ReactiveLimits reactiveLimits;
 
-    private LfStaticVarCompensatorImpl(StaticVarCompensator svc) {
+    private LfStaticVarCompensatorImpl(StaticVarCompensator svc, AbstractLfBus bus) {
         super(0);
         this.svc = svc;
-        double nominalV = svc.getTerminal().getVoltageLevel().getNominalV();
-        // min and  max reactive limit are calculated at nominal voltage
-        double minQ = svc.getBmin() * nominalV * nominalV;
-        double maxQ = svc.getBmax() * nominalV * nominalV;
         reactiveLimits = new MinMaxReactiveLimits() {
 
             @Override
             public double getMinQ() {
-                return minQ;
+                double v = bus.getV() * bus.getNominalV();
```

Comment:
The bus could be `null` (it means that the SVC is not connected). You are going to met a null pointer exception here. Same remark to get the nomial voltage, you have to get it from the terminal. I have to check but we could met problems in security analysis when some equipements are disconnected from the network. @geofjamg what is your opinion about this desgin ?

Reply:
Hello,
1/ For the moment, bus cannot be null because **LfStaticVarCompensatorImpl** is created by **AbstractLfBus** himself (AbstractLfBus : line 297).
```
    void addStaticVarCompensator(StaticVarCompensator staticVarCompensator, double scaleV, LfNetworkLoadingReport report) {
        if (staticVarCompensator.getRegulationMode() != StaticVarCompensator.RegulationMode.OFF) {
            add(LfStaticVarCompensatorImpl.create(staticVarCompensator, this),
                    staticVarCompensator.getRegulationMode() == StaticVarCompensator.RegulationMode.VOLTAGE,
                    staticVarCompensator.getVoltageSetPoint() * scaleV, -staticVarCompensator.getReactivePowerSetPoint(),
                    report);
        }
    }
```

2/ **bus.getNominalV()** returns a primitive type and thus cannot be null (but could be NaN ?)

3/ Considering `bus.getV() * bus.getNominalV();`
Here is code we can found in **AbstractLfBus** :
```
    public double getV() {
        return v / getNominalV();
    }
```
That's why I have to multiply by bus.getNominalV() to get v, is that correct ?